src/ILCompiler/ilrepro/repro.il

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
Could you add a copyright header and remove the ILDASM cruft (useless comments, useless labels for every line, useless `nop`'s, custom attributes, etc.)